eds4jinja2/builders/report_builder.py

```diff
@@ -40,11 +40,15 @@ def __init__(self, target_path, config_file=__DEFAULT_CONFIG_FILE__, output_path
 
         if output_path is not None:
             self.__DEFAULT_OUTPUT_FOLDER__ = pathlib.Path(output_path) / self.__DEFAULT_OUTPUT_FOLDER__
+        else:
+            self.__DEFAULT_OUTPUT_FOLDER__ = pathlib.Path(target_path) / self.__DEFAULT_OUTPUT_FOLDER__
```

Comment:
it will be more intuitive perhaps without the self.__DEFAULT_OUTPUT_FOLDER__

Reply:
de acord, nu cred că e nevoie de `self.__DEFAULT_OUTPUT_FOLDER__` dacă e indicat `output_path`